src/main/resources/env.schema.json

```diff
@@ -0,0 +1,195 @@
+    },
+    "OTP_TIMEZONE": {
+      "type": "string",
+      "default": "America/Los_Angeles",
+      "description": "The timezone identifier that OTP is using to parse dates and times. OTP will use the timezone identifier that it finds in the first available agency to parse dates and times."
+    },
+    "OTP_UI_NAME": {
+      "type": "string",
+      "default": "Trip Planner",
+      "description": "Config setting for linking to the OTP UI (trip planner)."
+    },
+    "OTP_UI_URL": {
+      "type": "string",
+      "default": "https://plan.example.com",
+      "description": "Config setting for linking to the OTP UI (trip planner)."
+    },
+    "MAXIMUM_PERMITTED_MONITORED_TRIPS": {
+      "type": "integer",
+      "default": "5",
+      "description": "The maximum number of saved monitored trips."
+    },
+    "MONGO_DB_NAME": {
+      "type": "string",
+      "default": "otp_middleware",
+      "description": "The name of the OTP Middleware Mongo DB."
+    },
+    "MONGO_HOST": {
+      "type": "string",
+      "default": "localhost:27017",
+      "description": "Mongo host address."
+    },
+    "MONGO_PROTOCOL": {
+      "type": "string",
+      "default": "mongodb",
+      "description": "Mongo DB protocol"
+    },
+    "NOTIFICATION_FROM_EMAIL": {
+      "type": "string",
+      "default": "noreply@email.com",
+      "description": "The from email address used in notification emails"
+    },
+    "NOTIFICATION_FROM_PHONE": {
+      "type": "string",
+      "default": "+15551234",
+      "description": "The from phone number used in notification SMSs"
+    },
+    "NOTIFICATION_TIME_FORMAT": {
+      "type": "string",
+      "default": "HH:mm",
+      "description": "The time format used in notification emails and SMSs."
+    },
+    "SENDGRID_API_KEY": {
+      "type": "string",
+      "default": "your-api-key",
+      "description": "Sendgrid settings available at: https://app.sendgrid.com/"
+    },
+    "SERVICE_DAY_START_HOUR": {
+      "type": "integer",
+      "default": "3",
+      "description": "Optional parameter for the hour (local time, 24-hr format) at which a service day starts. To make the service day change at 2am, enter 2. The default is 3am."
+    },
+    "SPARKPOST_KEY": {
+      "type": "string",
+      "default": "your-api-key",
+      "description": "Get Sparkpost key at: https://app.sparkpost.com/account/api-keys"
+    },
+    "TWILIO_ACCOUNT_SID": {
+      "type": "string",
+      "default": "your-account-sid",
+      "description": "Twilio settings available at: https://twilio.com/user/account"
+    },
+    "TWILIO_AUTH_TOKEN": {
+      "type": "string",
+      "default": "your-auth-token",
+      "description": "Twilio settings available at: https://twilio.com/user/account"
+    },
+    "VALIDATE_ENVIRONMENT_CONFIG": {
+      "type": "boolean",
+      "default": "true",
+      "description": "If set to true the env.yml file is validated against this schema."
```

Comment:
Maybe since this is optional a better description would be "If set to false, the validation of the env.yml file against this schema will be skipped."

Reply:
Updated.